Tools/Scripts/update-safer-cpp-expectations

```diff
@@ -160,15 +160,20 @@ def is_expected_file(expected_file):
                 expectations = f.read()
                 if line in expectations:
                     if not issues:
-                        print(f'{expected_file} has issues:')
+                        print(f'{expected_file} has the following issues:')
                     issues = True
                     print(f'- {project} {checker}')
     if not issues:
         print(f'{expected_file} has no known issues!')
+    else:
+        print('Follow this link for the latest results: https://build.webkit.org/#/builders/1288\n')
```

Comment:
I'd use `https://build.webkit.org/#/builders/Apple-Sonoma-Safer-CPP-Checks` instead.

Reply:
Oh! I didn't know you could link to the builder name :D